webpack.config.js

```diff
@@ -186,6 +180,15 @@ function makeExtensionConfig(development) {
       },
       /* indent width */ 2,
     ),
+
+    // Code splitting doesn't do any good in extensions
+    new webpack.optimize.LimitChunkCountPlugin({
+      maxChunks: 1,
+    }),
+
+    new webpack.DefinePlugin({
+      __ENV__: JSON.stringify("extension"),
```

Comment:
should this just be `"extension"`? what does `stringify` do here?

Reply:
This needs to be the JS-source-code to replace `__ENV__` with. If it was just `"extension"`, then we'd end up with code like `if (__ENV__ === "extension")` being converted to `if (extension === "extension")`, which won't work.

This could be written as `__ENV__: '"extension"'`, but that looks more confusing to me.

Reply:
ah! right makes sense.